themes/navy/layout/partial/share.swig

```diff
@@ -5,6 +5,6 @@
 <!-- Twitter follow button -->
 <a href="https://twitter.com/hexojs" class="twitter-follow-button" data-show-count="true">Follow @hexojs</a>
 <!-- Github Button -->
-<script async defer src="https://buttons.github.io/buttons.js"></script>
+<script async defer src="https://cdn.jsdelivr.net/npm/github-buttons@2.2.10/dist/buttons.min.js"></script>
```

Comment:
I don't think `async defer` is valid? Since it's not critical, should we just `defer` only?
@SukkaW

Reply:
~~I think the `defer` is no need.~~

IE11 not support `async`.

> https://caniuse.com/#search=async

Reply:
then we retain it then. the [installation guide](https://buttons.github.io/) does show "async defer".